fdbserver/TLogServer.actor.cpp

```diff
@@ -2323,6 +2320,12 @@ ACTOR Future<Void> tLogCommit(TLogData* self,
 		// Notifies the commitQueue actor to commit persistentQueue, and also unblocks tLogPeekMessages actors
 		logData->version.set(req.version);
 		if (SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST) {
+			for (Tag tag : tags) {
```

Comment:
Probably refactor this block into a member function of `logData`, to avoid repeats.